docs/mvvm/PuttingThingsTogether.md

```diff
@@ -454,6 +454,10 @@ The last missing piece is the UI for the post widget. As before, we've removed a
 
 Here we just have a header, with a `TextBlock` and an `ImageEx` control binding their `Text` and `Source` properties to the respective properties in our `Post` model, and a simple `TextBlock` inside a `ScrollViewer` that is used to display the (sample) content of the selected post.
 
+## Sample Application
+
+Sample application available [here](https://aka.ms/mvvmtoolkit).
```

Comment:
```suggestion
Sample application available [here](https://aka.ms/mvvmtoolkit/samples).
```

I created a more specific link earlier in case we move the main one to point elsewhere in the future.